src/api/libcellml/model.h

```diff
@@ -420,9 +420,7 @@ class LIBCELLML_EXPORT Model: public ComponentEntity
     /**
      * @overload
      *
-     * @brief Remove the import source with the given pointer.
-     *
-     * Remove the import source with the pointer @p imp.
+     * @brief Remove the import source at the given reference.
```

Comment:
Why has the "detailed" description been removed?

Reply:
Because it was the same as the brief one.  No point in duplicating it.

Reply:
Well, that breaks consistency (and not sure how it will look like in the documentation), but I have voiced my view, so I will leave it to @hsorby and @nickerso to comment.

Reply:
No point having a full description that is the same as the brief description (doxygen doesn't care, it will use the brief as needed). In my view the full description is only needed if the brief description is not sufficient.

Reply:
We are in a democracy, so... fine. :)